### DISCUSSION TITLE
Kitty inactive tab title.

### DISCUSSION THREAD

**Discussion by neelchakrabrty** [kovidgoyal/kitty#7815] [Ideas]
Hello, I was going through [kitty docs](https://sw.kovidgoyal.net/kitty/conf/#tab-bar), and found out that there is no way to set an inactive tab title. 
I think it might be an useful to keep track of tab numbers.

**Comment by kovidgoyal** [comment]
active_tab_title_template and tab_title_template together
allow specifying different templates for active and inactive tabs if
needed.

**Reply by neelchakrabrty** [reply]
Thank you, that fulfills my needs.
